### PR TITLE
Cob command merge

### LINKED ISSUE #62: Unify msh_upstart and upstart (cob_command)
At the moment each upstart configuration has a different setup.

### PR DESCRIPTION
**Pull request by ipa-fmw**
rebased with master
replaces #96
fixes #62 and #67

### CODE REVIEW

**Comment by ipa-fmw** [comment]
I'll merge this now as it cleans up multiple PRs

File: cob-pcs/cob-pcs-execute.sh [hunk 1]
```diff
@@ -9,15 +9,9 @@ if [ "$#" -lt 1 ]; then
...
-client_list="
-$robot_name-b1
-$robot_name-t1
-$robot_name-t2
-$robot_name-t3
-$robot_name-s1
-$robot_name-h1"
+# get pcs in local network
+IP=$(`hostname -I | awk '{print $1}'`)
+client_list=$(nmap --unprivileged $IP-98 --system-dns | grep report | awk '{print $5}')
```
Comment (ipa-nhg) [inline cob-pcs/cob-pcs-execute.sh]: please use ip addresses and not hostnames, the hostnames change depending on the network

`$(nmap --unprivileged $IP-98 --system-dns | grep report | awk '{print $6}' | sed 's/(//g;s/)//g')`

File: cob-pcs/cob-pcs-install.sh [hunk 2]
```diff
@@ -9,16 +9,9 @@ if [ "$#" -lt 1 ]; then
...
-client_list="
-$robot_name-b1
-$robot_name-t1
-$robot_name-t2
-$robot_name-t3
-$robot_name-s1
-$robot_name-h1"
+# get pcs in local network
+IP=$(`hostname -I | awk '{print $1}'`)
+client_list=$(nmap --unprivileged $IP-98 --system-dns | grep report | awk '{print $5}')
```
Comment (ipa-nhg) [inline cob-pcs/cob-pcs-install.sh]: please use ip addresses and not hostnames, the hostnames change depending on the network

`$(nmap --unprivileged $IP-98 --system-dns | grep report | awk '{print $6}' | sed 's/(//g;s/)//g')`

File: cob-pcs/cob-pcs-execute.sh (hunk 1 above)
Comment (ipa-fmw) [inline cob-pcs/cob-pcs-execute.sh]: done in https://github.com/ipa320/setup_cob4/pull/106

File: cob-pcs/cob-pcs-install.sh (hunk 2 above)
Comment (ipa-fmw) [inline cob-pcs/cob-pcs-install.sh]: done in https://github.com/ipa320/setup_cob4/pull/106

**Pull request merged**